CHANGELOG.md

```diff
@@ -4,6 +4,32 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.8.1](https://github.com/go-gitea/gitea/releases/tag/v1.8.1) - 2019-05-08
+* BUGFIXES
+  * Fix 404 when sending pull requests in some situations (#6871) (#6873)
+  * Enforce osusergo build tag for releases (#6862) (#6869)
+  * Don't post process commit summary in templates (#6868)
+  * Fix 500 when reviewer is deleted with integration tests (#6856) (#6860)
```

Comment:
I agree.

Reply:
```suggestion
  * Fix 500 when reviewer is deleted (#6856) (#6860)
```
I think if last part is removed it makes this more clear, as otherwise it sounds like tests may have been the problem.